client/src/containers/pages/locations/LocationDetail/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,267 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`containers/pages/Home renders without crashing 1`] = `
+<LocationDetail
```

Comment:
Is it possible to make this snapshot smaller?

Reply:
ok, let me try.